### PR TITLE
Fix depjs view

### PR DESCRIPTION
**Pull request by elemoine**
This reverts some of the changes introduced by #44 to fix the depjs view.

Please review.

### CODE REVIEW

File: pyramid_closure/views.py [hunk 1]
```diff
@@ -19,15 +16,19 @@ def depsjs(request):
...
-        else settings.get("pyramid_closure.roots_with_prefix")
-    )
-    roots_with_prefix = pairwise(roots_with_prefix)
+    for root in roots:
+        path_to_source.update(depswriter._GetRelativePathToSourceDict(root))
+
+    roots_with_prefix = pyramid_closure.get("roots_with_prefix") if \
+        pyramid_closure else \
+        settings.get("pyramid_closure.roots_with_prefix")
+    roots_with_prefix = aslist(roots_with_prefix or [])
```
Comment (sbrunner) [inline pyramid_closure/views.py]: in c2c.template case the roots_with_prefix will be a dictionary and this line will break it ...
`aslist({'aaa': 222}) => ['aaa']`

Comment (elemoine) [inline pyramid_closure/views.py]: See https://github.com/camptocamp/pyramid_closure/commit/e4e04409e65700c0257a0933bf485e2e9237b0e3#commitcomment-9497796 for another comment from @sbrunner.

**Comment by elemoine** [comment]
Thanks @sbrunner. Does it look better now?

**Comment by sbrunner** [comment]
Excellent :-)

**Pull request merged**